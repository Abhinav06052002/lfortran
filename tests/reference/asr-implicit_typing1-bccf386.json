{
    "basename": "asr-implicit_typing1-bccf386",
    "cmd": "lfortran --show-asr --implicit-typing --no-color {infile} -o {outfile}",
    "infile": "tests/implicit_typing1.f90",
    "infile_hash": "5f79fa57de4cbcd4a18a554a454dc2a5a2011d77fc4ea03114dede7b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-implicit_typing1-bccf386.stdout",
    "stdout_hash": "3b2a71313acaaac9c15052f737601a130808ec88db1d3bac240c1b8e",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}