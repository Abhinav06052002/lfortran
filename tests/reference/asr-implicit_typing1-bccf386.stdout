(TranslationUnit
    (SymbolTable
        1
        {
            x:
                (Program
                    (SymbolTable
                        2
                        {
                            a:
                                (Variable
                                    2
                                    a
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Real 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            integerx:
                                (Variable
                                    2
                                    integerx
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            l:
                                (Variable
                                    2
                                    l
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    x
                    []
                    [(Assignment
                        (Var 2 l)
                        (IntegerConstant 3 (Integer 4) Decimal)
                        ()
                    )
                    (Assignment
                        (Var 2 a)
                        (Cast
                            (IntegerConstant 3 (Integer 4) Decimal)
                            IntegerToReal
                            (Real 4)
                            (RealConstant
                                3.000000
                                (Real 4)
                            )
                        )
                        ()
                    )
                    (Assignment
                        (Var 2 integerx)
                        (IntegerConstant 12 (Integer 4) Decimal)
                        ()
                    )
                    (Print
                        (StringFormat
                            ()
                            [(Var 2 l)
                            (Var 2 a)
                            (Var 2 integerx)]
                            FormatFortran
                            (String -1 0 () PointerString)
                            ()
                        )
                    )]
                )
        })
    []
)
